zhaquirks/owon/Owon_PC321_Z.py

```diff
@@ -0,0 +1,99 @@
+"""QUIRK FOR OWON PC321 Z"""
```

Comment:
Let's not put this in all caps.
Also, add a dot at the end. The automated checks don't test for this anymore, but I'd still do it.

Reply:
OK modified